webapp/package.json

```diff
@@ -23,5 +23,8 @@
     "simple-git": "^3.20.0",
     "yaml": "^2.3.3",
     "zod": "^3.22.4"
+  },
+  "devDependencies": {
+    "eslint-config-next": "13.5.6"
```

Comment:
I think it is better to keep dev dependencies in the root

Reply:
If I remember correctly you explained to me that the reason for this preferences is that npm audit complains if one has dependencies with vulnerable devDependencies. I cannot find any trace of that in npm docs, yarn docs or on google. Can you explain again why we prefer devDependencies in the root?

Reply:
We discussed this on slack and found two reasons to keep devDependencies in the root:
* Tools like npm audit automatically omits devDependencies of the root if run in the sub-project which is convenient as production dependencies are more important and you can still audit your devDepenendencies if you run the auditing tool in the root.
* If we would publish the sub-project, we can do so without publishing which devDependencies we used. We see no use for dependents to review our devDependencies.

Reply:
I moved the devDependency back to root in e1b254706511ee98440df7a7874a557d695d4ef4